README.md

```diff
@@ -14,11 +16,13 @@ If you new to Java, or to Software Engineering, you should simply try to downloa
 
 Otherwise, I highly recommend you to manage your Java installations using the awesome https://sdkman.io/
 
-## Quickstart
+### Access Nubank's Github 
```

Comment:
There is nothing here tying the tool up to Nubank's Github. Anyone who has a Github account can use this tool.

```suggestion
### Access Github 
```